platform/android/java/lib/src/org/godotengine/godot/GodotLib.java

```diff
@@ -191,6 +191,22 @@
 	 */
 	public static native void method(String p_sname, String p_name, String p_ret, String[] p_params);
 
+	/**
+	 * Used to complete registration of the {@link org.godotengine.godot.Godot.SingletonBase} instance's methods.
```

Comment:
Here and below, can you specify whether the methods should be invoked on the Android main thread, or on the GL thread, or whether it doesn't matter.

Reply:
I didn't test it very well, but with Firebase it works without any care about the current thread.  I'm now updating other my plugins to make sure that all works.